Loadouts/nato_mx_mcam.hpp

```diff
@@ -0,0 +1,228 @@
+// Author: BWMF
+// Description: NATO: MX Series - MCAM
+
+#include "undef.hpp" // Reset defines
+
+// Rifle
+#define RIFLE "arifle_MX_F"
+#define RIFLE_MAG "30Rnd_65x39_caseless_mag:8","30Rnd_65x39_caseless_mag_Tracer:2"
+#define RIFLE_ATTACHMENTS "optic_Aco","acc_pointer_IR"
+#define AAR_ATTACHMENTS RIFLE_ATTACHMENTS
+#define ALT_OPTICS "rhsusf_acc_compm4","rhsusf_acc_eotech_xps3","CUP_optic_CompM2_Black","CUP_optic_TrijiconRx01_black","BWA3_optic_RSAS","RKSL_optic_RMR_MS19"
+// GL Rifle
+#define GLRIFLE "arifle_MX_GL_F"
+#define GLRIFLE_MAG RIFLE_MAG
+#define GLRIFLE_MAG_SMOKE "1Rnd_Smoke_Grenade_shell:2","1Rnd_Smoke_Grenade_shell:2"
+#define GLRIFLE_MAG_HE "1Rnd_HE_Grenade_shell:5"
+#define GLRIFLE_MAG_FLARE "UGL_FlareYellow_F:4"
+// Carbine
+#define CARBINE "arifle_MXC_F"
+#define CARBINE_MAG RIFLE_MAG
+// AR
+#define AR "arifle_MXM_F"
```

Comment:
The AR right now can only load 30 round magazines. If you change it to the MX-LSW the AR can use 100 round magazines